### ISSUE TITLE
Support for DataTypes.VIRTUAL constructor?

### ISSUE THREAD

**Issue by ThomasGHenry** [davesag/sequelize-test-helpers#161]
Hey! Thanks for this!

I noticed that mocking models that instantiate virtual fields, like this...
```
type: new DataTypes.VIRTUAL(DataTypes.BOOLEAN)
```
https://sequelize.org/master/class/lib/data-types.js~VIRTUAL.html

...yields this error...
```
TypeError: DataTypes.VIRTUAL is not a constructor
```

 I reproduced it here: https://github.com/ThomasGHenry/sequelize-test-helpers/blob/develop/test/models/Simple.js

Run `npm test` to see the error.

Is this expected?

Thanks!

**Comment by davesag** [comment]
Hi @ThomasGHenry thanks for the feedback.

I'll aim to take a look at this over the weekend.  Else, if you are interested, feel free to create a pull request.

**Comment by ThomasGHenry** [comment]
Thanks! Not urgent. I worked around it by pulling in the official data types.

**Comment by davesag** [comment]
@ThomasGHenry fixed in Version 1.3.0 which has just been released.  Thanks for your feedback.
